src/changelog/buildpacks/_posts/2024-05-24-nodejs-18.20.3-22.2.0.md

```diff
@@ -0,0 +1,11 @@
+---
+modified_at: 2024-05-24 12:00:00
+title: 'Node.js - Support for version 18.20.3 and 22.2.0'
+github: 'https://github.com/Scalingo/nodejs-buildpack'
+---
+
+- Support for Node.js `18.20.3` and `22.2.0`
```

Comment:
question: Is one of these versions the new default? If yes, maybe highlight it here? Otherwise LGTM.

Reply:
According to our documentation, the latest `20.x` version is the default one.
Checking the buildpack code, it seems it's indeed the case: https://github.com/Scalingo/nodejs-buildpack/blob/550ad0c79fb85fb35470d2bef00ae1eada26f69a/lib/binaries.sh#L74

So nothing to do here :) Thanks for making sure <3